### ISSUE TITLE
Path Error

### ISSUE THREAD

**Issue by yhgcn** [JacobBrandt/timeline_heatmap#1]
Run in kibana 5.2.2, there is an error：
 log   [09:37:33.751] [fatal] Error: Optimizations failure.
    ERROR in ./plugins/timeline_heatmap/public/timeline_heatmap_vis.js
    Module build failed: SyntaxError: ……

==============================================
```
vi public/timeline_heatmap_controller.js
#    var TemplateVisType = Private(require("ui/template_vis_type/TemplateVisType"));
    var TemplateVisType = Private(require("ui/template_vis_type/template_vis_type"));
#    var Schemas = Private(require('ui/Vis/Schemas'));
    var Schemas = Private(require('ui/vis/schemas'));

vi public/timeline_heatmap_controller.js
#  const Binder = require('ui/Binder');
  const Binder = require('ui/binder');
```

**Comment by JacobBrandt** [comment]
There's a 5.1 branch for Kibana 5 which as you noted has different paths.  There were a few more breaking changes in Kibana 5 release hence the reason for a separate branch.  The master branch isn't really meant for latest Kibana versions.  I'll update the branches so that it's clear what version of Kibana they are for.

**Comment by JacobBrandt** [comment]
I merged 5.1 branch to master and updated the README.md.  It now includes links to the two different branches of the plugin corresponding to which version of Kibana they support.
